Upstream r81599 broke added a field in the middle of struct ast_filestream.
This resulted in breaking the ABI and having crashes on plugins compiled with
older versions of asterisk-dev (e.g. format_mp3)

This moves the new field in the end of the struct.

Should be forwarded upstream, if not too late.

 -- Faidon Liambotis <paravoid@debian.org>

--- asterisk-1.4.12.1~dfsg.orig/include/asterisk/file.h
+++ asterisk-1.4.12.1~dfsg/include/asterisk/file.h
@@ -129,11 +129,11 @@ struct ast_filestream {
 	int lastwriteformat;
 	int lasttimeout;
 	struct ast_channel *owner;
-	const char *orig_chan_name;
 	FILE *f;
 	struct ast_frame fr;	/* frame produced by read, typically */
 	char *buf;		/* buffer pointed to by ast_frame; */
 	void *private;	/* pointer to private buffer */
+	const char *orig_chan_name;
 };
 
 #define SEEK_FORCECUR	10
